frontend/src/admin/components/userList/userList.tsx

```diff
@@ -0,0 +1,45 @@
+import '../userList/userlist.scss'
+import React, { useState } from 'react';
+import Users from "../../users.json";
+const UserList = () => {
+    const [active, setActive] = useState();
+    return (
+        <section className={"userlist"}>
+            <div className="userlist__header">
+                <h2>Users</h2>
+                <button className={"userlist__header__button"}>
+                    <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
+                        <g id="Group_71" data-name="Group 71" transform="translate(-448 -83)">
+                            <g id="Ellipse_1" data-name="Ellipse 1" transform="translate(448 83)" fill="none" stroke="#fff">
+                                <circle cx="16" cy="16" r="16" stroke="none"/>
+                                <circle cx="16" cy="16" r="15" fill="none"/>
+                            </g>
+                            <g id="Group_70" data-name="Group 70">
+                                <circle id="Ellipse_2" data-name="Ellipse 2" cx="2.5" cy="2.5" r="2.5" transform="translate(454 97)" fill="#fff"/>
+                                <circle id="Ellipse_3" data-name="Ellipse 3" cx="2.5" cy="2.5" r="2.5" transform="translate(462 97)" fill="#fff"/>
+                                <circle id="Ellipse_4" data-name="Ellipse 4" cx="2.5" cy="2.5" r="2.5" transform="translate(470 97)" fill="#fff"/>
+                            </g>
+                        </g>
+                    </svg>
+                </button>
+            </div>
+            <div className="userlist__buttons">
+                <div className="userlist__buttons__userbutton">
+                    {Users.users.map(user => (
+                        <button
+                            key={user.id}
+                            className={`${active == user}`}
+                            onClick={() => setActive(user)}
```

Comment:
It is sufficient to use the user ID for referencing the active user.

Reply:
But there is also the philosohy that when writing object oriented (which I strongly suggest we do) we don't split our objects out into attributes whenever we call a function but instead send along the entire object to the function.

Offcoure, it should be evaluated on a case by case basis.

Reply:
The key is for now the user id

Reply:
@LindvedKrvang Good point. I am fine with either solution.